src/util/request-manager.js

```diff
@@ -195,7 +196,6 @@ export default class RequestManager {
 
   _getRequestModule(): RequestModuleT {
     if (!this._requestModule) {
-      const request = require('request');
```

Comment:
I think this require has been inlined not because of lazy import, but rather so that it can be mocked by Jest (if hoisted to the top of the file, each invocation will reuse the same `request` instance than the previous ones, ignoring any new mock registered through Jest).

I believe it's the root cause for your failing tests.